common/src/main/java/org/apache/uniffle/common/compression/Codec.java

```diff
@@ -72,5 +74,6 @@ public enum Type {
     ZSTD,
     NOOP,
```

Comment:
The type of NOOP is available for this case, although we should specify it explicitly

Reply:
NOOP type exists bytes array copy.

Reply:
How about making the NoOp to bypass all the copy  ?

Reply:
> Codec.decompress requires a new buffer, which is not necessary for non compression.

As commented above, this seems unavoidable. Should we care about this?